ios/Runner/AppDelegate.swift

```diff
@@ -37,4 +45,22 @@ let apnTokenKey = "apnToken"
         // Save the token to use in the future
         UserDefaults.standard.set(token, forKey: apnTokenKey)
     }
+    
+    override func userNotificationCenter(_ center: UNUserNotificationCenter,
+                                         willPresent notification: UNNotification,
+                                         withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
+        let userInfo = notification.request.content.userInfo
+        twakeApnChannel?.invokeMethod("willPresent", arguments: userInfo)
```

Comment:
just curious, when this method get called ?

Reply:
check should show a noti when app in foreground
```
Asks the delegate how to handle a notification that arrived while the app was running in the foreground.
```
https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649518-usernotificationcenter